deploy/runtools/run_farm_deploy_managers.py

```diff
@@ -24,6 +27,8 @@
 
 rootLogger = logging.getLogger()
 
+_RFC_3986_PATTERN = re.compile(r"^[A-Za-z][A-Za-z0-9+\-+.]*://")
```

Comment:
```suggestion
# from  https://github.com/pandas-dev/pandas/blob/96b036cbcf7db5d3ba875aac28c4f6a678214bfb/pandas/io/common.py#L73
_RFC_3986_PATTERN = re.compile(r"^[A-Za-z][A-Za-z0-9+\-+.]*://")
```

@davidbiancolin is this the kind of ref you are wanting?

Reply:
Perfect.